docs/userguide/parameters.md

```diff
@@ -1,6 +1,15 @@
+[`SearchSpace`]: baybe.searchspace.core.SearchSpace
+[`Constraint`]: baybe.constraints.base.Constraint
+[`SubstanceParameter`]: baybe.parameters.categorical.SubstanceParameter
```

Comment:
(related to other thread) @Hrovatin the link to the substance parameter is incorrect here, please fix (unsure if this already fixes the other issue, still invstigating)

Reply:
Done it myself, hence resolving.